composer.json

```diff
@@ -26,6 +26,12 @@
         "phpunit/phpunit": "~6.0.0",
         "mikey179/vfsstream": "~1.6"
     },
+    "repositories": {
+        "distributionPackages": {
```

Comment:
Wait… why give it a name at all? I never did that, so far.

Reply:
Also, using JSON object notation (not an array) does not guarantee order. See https://getcomposer.org/doc/04-schema.md#repositories